nltk/align/ibm1.py

```diff
@@ -118,6 +118,8 @@ def align(self, align_sent):
             # Initialize the maximum probability with Null token
             max_align_prob = (self.probabilities[en_word][None], None)
             for i, fr_word in enumerate(align_sent.mots):
+                max_align_prob = (max_align_prob[0], max_align_prob[1] if
+                                  max_align_prob[1] else 0)
```

Comment:
This probably isn't what you want, since it will break the comparison to None a few lines down. And it will still blow up lthe next statement in the case when max_align_prob[0] == self.probabilities[en_word][fr_word]

There is also another unrelated issue with this align method in handling OOV words (out of vocabulary), because there is no smoothing. Perhaps there needs to be a rethink about how to handle words that were not seen during training.